db/db_compaction_test.cc

```diff
@@ -3300,11 +3300,14 @@ TEST_P(DBCompactionTestWithParam, CompressLevelCompaction) {
   GenerateNewFile(&rnd, &key_idx);
   ASSERT_EQ("1,4,8", FilesPerLevel(0));
 
-  ASSERT_EQ(matches, 12);
```

Comment:
@cbi42 the check for `IsTrivialMove` inside my `GetNumberCompactionSortedRuns` method messes up the count for `InputCompressionMatchesOutput`. I did not dig into why the original test values were the way they are, but I do not think my PR would break this functionality. For those who do understand, I updated the expected value to be the original expected value plus what I determined was added by `GetNumberCompactionSortedRuns`.

Not sure if you had to work with this test case before. Let me know if this is problematic or the comments can be reworded